src/main/scala/com/softwaremill/react/kafka/KafkaActorSubscriber.scala

```diff
@@ -23,17 +22,12 @@ private[kafka] class KafkaActorSubscriber[T](
       processElement(element.asInstanceOf[T])
     case ActorSubscriberMessage.OnError(ex) =>
       handleError(ex)
-    case ActorSubscriberMessage.OnComplete =>
+    case ActorSubscriberMessage.OnComplete | "Stop" =>
```

Comment:
Doesn't need to block this PR, but maybe it's worth discussing it with the Akka guys? Seems like something that shouldn't be necessary.